ipatests/test_webui/test_sysaccount.py

```diff
@@ -0,0 +1,218 @@
+# Copyright (C) 2025  Red Hat
+# see file 'COPYING' for use and warranty information
+"""Tests for FreeIPA system accounts webui functionality"""
+
+import os
+
+from ipatests.test_webui.ui_driver import UI_driver
+from ipatests.test_webui.ui_driver import screenshot
+import ipatests.test_webui.data_sysaccount as sysaccount
+import ipatests.test_webui.test_rbac as rbac
+import pytest
+from ipatests.pytest_ipa.integration import tasks
+from ipapython.ipaldap import realm_to_serverid
+
+try:
+    from selenium.webdriver.common.by import By
+except ImportError:
+    pass
+
+
+@pytest.mark.tier1
+class test_sysaccount(UI_driver):
+    """
```

Comment:
Can you add a test job in prci definitions for this new class? No need in 389ds, pki and sssd but the other files should launch the new test.